docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-definition/modal-content-snippet.tsx

```diff
@@ -0,0 +1,127 @@
+import CgdpJsx from '../../core/jsx';
+
+// The array of media in the response
+// This is where the images are for the definition
+type ResponseMediaArrayObject = {
```

Comment:
It would be better if this came from the client since the client defines this shape.